css/style.css

```diff
@@ -0,0 +1,220 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  font: 12px/18px "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  background-color: #eee;
+  color: #333;
+  margin: 0;
+  background-image: url("../images/bg-img.jpg");
+  background-repeat: no-repeat;
+  background-size: cover;
+  background-position: 50% 180%;
+  width: 1349px;
+}
+
+.header-container {
+  background-color: transparent;
+  width: 1320px;
+  display: flex;
+  justify-content: center;
+  height: 550px;
+}
+
+.nav-container {
+  height: 35px;
+  width: 900px;
+  border-radius: 5px;
+  margin: 0 auto;
+  background: rgb(128, 128, 128);
+  background: linear-gradient(rgb(128, 128, 128) 48%, rgb(114, 112, 112) 50%);
+  display: flex;
+  margin-top: 11%;
```

Comment:
Decrease the `margin-top` to match the original.

### Original

![Annotation 2020-06-21 113119](https://user-images.githubusercontent.com/57726348/85217898-c7736300-b3b2-11ea-9aab-844048444bc2.jpg)